examples/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "composite": true,
+    "jsx": "react-jsx",
+    "outDir": "./dist/"
+  },
+  "include": ["./**/*", "./global.d.ts"]
```

Comment:
`./global.d.ts` is automatically part of `./**/*`

Reply:
True, I'll doublecheck if it can be removed, I think it didn't work when I made this change.